### DISCUSSION TITLE
monika isn't seeing any of the gifts i put in the files for her

### DISCUSSION THREAD

**Discussion by uwumel** [Monika-After-Story/MonikaModDev#9629] [QA - tech support + general] [answered]
i find some spritepacks on reddit, download them, and put them in the right file directories, and she doesn't recognize them? it used to work before, but now i just don't know what happened. is there a new way for giving gifts?

**Comment by Booplicate** [comment, marked as answer]
Either the spritepacks are invalid or you're installing them incorrectly. You can try following [this guide](https://github.com/Monika-After-Story/MonikaModDev/wiki/Adding-Sprite-Objects#troubleshooting-installation).
